test/php/gutenberg-coding-standards/composer.json

```diff
@@ -8,7 +8,7 @@
 		"static analysis",
 		"Gutenberg"
 	],
-	"license": "MIT",
+	"license": "GPL-2.0-or-later",
```

Comment:
Not sure how important this is but maybe set this to `GPL-2.0-or-later, MPL 2.0` to match https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md ?

Reply:
@azaozz 
The intention was to match Gutenberg's composer.json file to avoid confusion or ambiguity. Gutenberg's composer.json file currently specifies GPL-2.0-or-later as the license.

https://github.com/WordPress/gutenberg/blob/trunk/composer.json#L4

But yes, LICENSE.md says Gutenberg is dual-licencsed.

Reply:
If you think that's a blocker, I don't mind updating `composer.json` to use both licenses.

Reply:
Nah, don't think it's that important, moreover it matches what's in the parent directory.